pkg/xen-tools/Dockerfile

```diff
@@ -26,10 +26,12 @@ ENV BUILD_PKGS \
     gettext iasl util-linux-dev ncurses-dev glib-dev \
     pixman-dev libaio-dev yajl-dev argp-standalone \
     linux-headers git patch texinfo tar libcap-ng-dev \
-    attr-dev flex bison cmake libusb-dev zstd-dev
+    attr-dev flex bison cmake libusb-dev zstd-dev \
+    python3 build-base ninja
```

Comment:
ninja is replaced by samirai in Alpine. same binary different name https://pkgs.alpinelinux.org/package/edge/main/x86_64/samurai